src/ToggleIcon.js

```diff
@@ -13,7 +13,8 @@ const styles = {
     width: 24,
     height: 24,
     position: 'relative',
-    display: 'inline-block'
+    display: 'inline-block',
+    transition: 'clip-path 550ms cubic-bezier(0.4, 0.0, 0.2, 1), -webkit-clip-path 550ms cubic-bezier(0.4, 0.0, 0.2, 1)'
```

Comment:
Adding this broke the animation, which is the sole purpose of this component. :smile:

Reply:
Oops, sorry! Not sure how that ended up there, weird that I didn't noticed it. I'm sure I tested it a couple of times :disappointed: